components/dashboard/src/contexts/FeatureFlagContext.tsx

```diff
@@ -31,13 +33,15 @@ const FeatureFlagContextProvider: React.FC = ({ children }) => {
     const team = getCurrentTeam(location, teams);
     const [showPersistentVolumeClaimUI, setShowPersistentVolumeClaimUI] = useState<boolean>(false);
     const [showUsageView, setShowUsageView] = useState<boolean>(false);
+    const [showUseLastSuccessfulPrebuild, setShowUseLastSuccessfulPrebuild] = useState<boolean>(false);
 
     useEffect(() => {
         if (!user) return;
         (async () => {
             const featureFlags: FeatureFlagConfig = {
                 persistent_volume_claim: { defaultValue: true, setter: setShowPersistentVolumeClaimUI },
                 usage_view: { defaultValue: false, setter: setShowUsageView },
+                showUseLastSuccessfulPrebuild: { defaultValue: false, setter: setShowUseLastSuccessfulPrebuild },
```

Comment:
What's the current pattern we want to use for Feature Flag names? I thought it'd be `snake_case`? But might be wrong! :thinking:

Reply:
There is [a debate](https://gitpod.slack.com/archives/C03NTLVU134/p1665995070215639) (internal) about this currently 🙂